cartographer_grpc/local_trajectory_uploader.h

```diff
@@ -51,16 +51,25 @@ class LocalTrajectoryUploader {
       const cartographer::mapping::proto::TrajectoryBuilderOptions&
           trajectory_options);
   void FinishTrajectory(int local_trajectory_id);
+
+  // Enqueue an Add*DataRequest message to be uploaded.
   void EnqueueDataRequest(
       std::unique_ptr<google::protobuf::Message> data_request);
 
+  SensorId GetLocalSlamResultSensorId(int local_trajectory_id) const {
```

Comment:
I am torn whether we should use the local or global trajectory ID here. This ID will appear potentially in error messages, which could occur both on the agent and in the cloud. I guess since the cloud has a lot of trajectories it is more helpful to find a reference that allows you to indentify which of the many trajectories are causing problems. On the agent we will have 1 or 2 only. WDYT?

Reply:
We can only use the local one because it is already required to pass the expected sensors to uplink->AddTrajectory before we know the uplink_trajectory_id.

Reply:
Right, of course.